src/services/listen-service/qrd.ts

```diff
@@ -1,10 +1,12 @@
-import { Collection, Message, Channel } from "discord.js";
+import { Collection, Message } from "discord.js";
 import { BotCommand } from "../../bot/parseCommand";
 import { Option, orDefault, parseToNumber } from "../../util";
 import { deepApology, speak } from "../../bot/speak";
 import compareAsc from "date-fns/compareAsc";
+const openai = require('openai');
+
+openai.apiKey = 'your-openai-api-key';
```

Comment:
The way we'll handle this configuration is to inject it via the environment. On app start, config is read from the environment like so: https://github.com/dpchamps/butler-bot/blob/master/src/config.ts#L30-L52.

So we'd add

```typescript
OPENAI_KEY: String
```

Config is passed into the listenService, so probably the best place to to add this is here: https://github.com/dpchamps/butler-bot/blob/master/src/services/listen-service/listen-service.ts#L89

something like 

```typescript
// top of file
import OpenAI from 'openai';

// line 88.5
const openai = new OpenAI({
  apiKey: config.OPENAI_KEY
});
```